sdk/search/azure-search-documents/swagger/src/main/java/SearchIndexCustomizations.java

```diff
@@ -114,21 +113,6 @@ private void customizeSearchOptions(ClassCustomization classCustomization) {
 //                "this.scoringParameters.stream().map(ScoringParameter::new).collect(java.util.stream.Collectors.toList())");
     }
 
-    private void customizeSearchQueryVector(ClassCustomization classCustomization) {
```

Comment:
We'll still want this but we just need to change this to modify `VectorQuery` instead of `SearchQueryVector`. Also, we need to update the body to null check `fields` with `this.fields = (fields == null) ? null : String.join(",", fields);`